### DISCUSSION TITLE
Default Theme - Search

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#4322] [General]
The new Search capability was added to the default Oqtane theme. I would like to get some feedback on how it should be integrated in the UI...

Option 1:

![image](https://github.com/oqtane/oqtane.framework/assets/4840590/1eef1296-10d5-44d4-b28c-cd3b919daa8d)

![image](https://github.com/oqtane/oqtane.framework/assets/4840590/98e58ad3-fc4d-49a8-8cef-02d07e551ec0)

Option 2:

![image](https://github.com/oqtane/oqtane.framework/assets/4840590/142395c2-9d06-4f39-8731-479d9b2054a8)

![image](https://github.com/oqtane/oqtane.framework/assets/4840590/5b036a49-176e-43c6-8598-3e19f9c80062)

**Comment by mdmontesinos** [comment]
I'd say option 2. Login is usually furthest to the right, and logout should be next to the other actions buttons (edit, config, ...)
